WinImageBuilder.psm1

```diff
@@ -537,6 +547,23 @@ function Add-VirtIODrivers {
 }
 
 function Add-VirtIODriversFromISO {
+    <#
+    .SYNOPSIS
+     This function allows you to add VirtIO drivers to your Windows image, from an ISO file.

```

Comment:
please keep a concise non personal communication pattern:

This function adds VirtIO drivers from a given ISO path to a mounted Windows VHD image.